### PR TITLE
Fixed unexpected errors with the email load pipeline

### LINKED ISSUE #76: Fix Issue with Email Pipeline
Fix the error with email script as it raises an error if `price_threshold` is a `NoneType`.

### PR DESCRIPTION
**Pull request by BerkayDur**
The email load pipeline had unexpected errors, due to `previous_price` being `null`, and `price_threshold` being `null`. These are now fixed. closes #76

### CODE REVIEW

File: ETL/Email/conftest.py
Comment (danfh00) [inline ETL/Email/conftest.py:1]: Great use of a conftest file and fixtures to make the test file more readable.

File: ETL/Email/email_service.py [hunk 1]
```diff
@@ -281,8 +322,8 @@ def send_emails(
         'product_id' : 1,
         'url' : 'https://www.asos.com/nike-running/nike-running-juniper-trail\
 -2-gtx-trainers-in-grey/prd/205300355#colourWayId-205300357',
-        'current_price' : 83.99,
-        'previous_price' : 104.99,
+        'current_price' : 89.99,
+        'previous_price' : 89.99,
         'is_on_sale' : True,
```
Comment (danfh00) [inline ETL/Email/email_service.py]: Is this sample data still required?

File: ETL/Email/helpers.py
Comment (danfh00) [inline ETL/Email/helpers.py:1]: Great use of helpers file to make the main file easier to understand

File: ETL/Email/test_email_service.py [hunk 2]
```diff
@@ -104,44 +104,37 @@ def test_get_customer_information_type_error_3(mock_create_single_insert_format_
...
+    '''testing filter_merged_table with valid DataFrame values.'''
+    assert filter_merged_table(fake_merged_data).sort_values('price_threshold').reset_index(drop=True).equals(pd.DataFrame([
+        {'price_threshold': None, 'current_price': 18.96, 'previous_price': 19.98, 'is_on_sale': True},
+        {'price_threshold': 19.99, 'current_price': 18.91, 'previous_price': None, 'is_on_sale': False},
+        {'price_threshold': 19.98, 'current_price': 18.90, 'previous_price': None, 'is_on_sale': True},
+        {'price_threshold': 19.96, 'current_price': 18.88, 'previous_price': 20.99, 'is_on_sale': True},
+        {'price_threshold': 19.95, 'current_price': 18.87, 'previous_price': 20.98, 'is_on_sale': False},
+    ]).sort_values('price_threshold').reset_index(drop=True))
+
+@pytest.mark.parametrize('fake_data', [[{'product_id': 1, 'price_threshold': 10.0}],
```
Comment (danfh00) [inline ETL/Email/test_email_service.py:117]: Great use of paramaterize to execute many tests

**Review by danfh00 — APPROVED** [review]

**Review by danfh00 — DISMISSED** [review]

File: ETL/Email/email_service.py (hunk 1 above)
Comment (BerkayDur) [reply ETL/Email/email_service.py]: No, I will remove it!

File: ETL/Email/email_service.py
Comment (danfh00) [inline ETL/Email/email_service.py:1]: Great, thanks for removing that

**Review by danfh00 — DISMISSED** [review]
Great, looks simpler without the sample data

File: ETL/Email/email_service.py [hunk 3]
```diff
@@ -4,6 +4,7 @@
 from os import environ as CONFIG
 from datetime import datetime
```
Comment (A-Faris) [inline ETL/Email/email_service.py]: Unused import, destroy this

**Review by danfh00 — DISMISSED** [review]

**Review by A-Faris — APPROVED** [review]
Thanks, looks good

**Pull request merged**